HPXMLtoOpenStudio/resources/util.rb

```diff
@@ -321,3 +321,33 @@ def self.fetch(uri_str, outfile, agent = 'curl/7.43.0', max_attempts = 10, timeo
     raise 'Too many http redirects' if attempts == max_attempts
   end
 end
+
+class UnitTestTools
```

Comment:
@shorowit what do you think about this class name? Any suggestions?

Reply:
To be honest, I was picturing this method in a new test.rb file under /tests, as opposed to under /resources, since I can't imagine it would ever be used outside tests. No strong preference about class name; not sure it really needs a class for organization considering it's just a single method.

Reply:
That makes sense. Will create a util.rb under tests. I agree that this method doesn't need to be in a class.